### ISSUE TITLE
Predictor.py arguments

### ISSUE THREAD

**Issue by alejandrojapkin** [KaroDievas/car-sound-classification-with-keras#3]
According to the code are these:
img_path = sys.argv[1]
weights = sys.argv[2]
So not just the img path as provided in the readme. Please provide a full example.

**Comment by KaroDievas** [comment]
Hi, 

I am currently developing this project (the main issue now that my model is overfitting, After ~20 epochs I get zero loss), so readme will be provided later. 
As I am trying to write self documented code, you can easily understand that first parameter is path to image which should be recognized and the second parameter is weights which was generated at training step.

**Comment by alejandrojapkin** [comment]
The full command line parameters of a working example for training and prediction, within a specific instance, which you've tested successfully for the present commit, is necessary to test and provide feedback on this experiment.  
It'd be good to start with something that works for sure.

**Comment by KaroDievas** [comment]
Documentation is updated. Closing ticket
